contrib/pc-sanity/bin/platform_userguide_test

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+failed()
+{
+    [ -n "$1" ] && echo "$1"
+    echo "$0 failed!"
```

Comment:
How about using `>&2 echo` to print the errors to the `stderr` which makes more perfect sense.

Reply:
thanks!